dashboard/src/components/Header.vue

```diff
@@ -3,76 +3,92 @@ import { openDialog } from "vue3-promise-dialog";
 
 import AboutDialogVue from "@/components/AboutDialog.vue";
 import Breadcrumb from "@/components/Breadcrumb.vue";
+import InstitutionLogo from "@/components/InstitutionLogo.vue";
 import { useLayoutStore } from "@/stores/layout";
 import IconInfo from "~icons/clarity/info-standard-solid";
 import IconMenu from "~icons/clarity/menu-line";
 
 const layoutStore = useLayoutStore();
 
 const showAbout = async () => await openDialog(AboutDialogVue);
+
+const institution: { logo: string; name: string; url: string } = {
+  logo: import.meta.env.VITE_INSTITUTION_LOGO,
+  name: import.meta.env.VITE_INSTITUTION_NAME,
+  url: import.meta.env.VITE_INSTITUTION_URL,
+};
 </script>
 
 <template>
   <header class="bg-white border-bottom sticky-top">
     <nav class="navbar navbar-expand-md p-0">
-      <!-- Open offcanvas button, only visible in sm. -->
-      <button
-        type="button"
-        class="navbar-toggler btn btn-link text-decoration-none p-3"
-        data-bs-toggle="offcanvas"
-        data-bs-target="#menu-offcanvas"
-        aria-controls="menu-offcanvas"
-        aria-label="Open navigation"
-      >
-        <IconMenu
-          class="text-dark mx-1"
-          style="font-size: 1.5em"
-          aria-hidden="true"
-        />
-      </button>
+      <div class="container-fluid">
```

Comment:
The `container-fluid` adds some padding to the sides we don't want. We could use the specific flex classes we want, or just add `px-0` to remove it.

Reply:
I added the "container-fluid" class based on the examples at https://getbootstrap.com/docs/5.3/components/navbar/.  I'm fine to remove it, but I wish our use of Bootstrap classes didn't feel so arbitrary, I really have a hard time figuring out what classes to use. :(

Reply:
I've removed the div altogether in the latest commit.